ETC.tsv

```diff
@@ -3199,7 +3199,7 @@ ETC_20150317_003198	Destroy the Flag of the Vubbe tribe!
 ETC_20150317_003199	Installing bomb
 ETC_20150317_003200	Secret passage of Vubbes
 ETC_20150317_003201	Fight back the Vubbes and install a bomb in the secret passage
-ETC_20150317_003202	Monsters are trying to get rid of the seal! Get rid of them!
+ETC_20150317_003202	Monsters are trying to attack the seal! Defeat them!
```

Comment:
Not necessarily attack, get rid of could imply they are trying to remove it by other means besides attacking it. I suggest changing it to: Monster are trying to remove the seal! Defeat them!